torch/_decomp/decompositions.py

```diff
@@ -2245,6 +2373,9 @@ def one_layer_lstm(inp, hidden, params, has_biases, reverse=False):
         hx = hy
         cx = cy
 
+    if reverse:
+        step_output.reverse()
+
```

Comment:
A bit confused about what's going on here. Why is this necessary for LSTM? (It isn't in the RNN decomp). If it is necessary, then should this be in the previous PR?

Reply:
Yeah so what's happening here is that all the decompositions are using `_rnn_helper`

But that was baking in an assumption that this was going to be a tensor. This basically replaces [L2141](https://github.com/pytorch/pytorch/pull/91281/files#diff-d25ad317caf41c3fdf1de3bcfc730102c9e4a3e68639559017ae45cb0bc26911L2141) because I didn't really think ahead to PackedSequences when I was doing this

Reply:
Thanks for pointing that out, I missed that line